base/uk.ac.stfc.isis.ibex.dae/src/uk/ac/stfc/isis/ibex/dae/spectra/SpectrumYAxisTypes.java

```diff
@@ -0,0 +1,26 @@
+package uk.ac.stfc.isis.ibex.dae.spectra;
+
+import uk.ac.stfc.isis.ibex.ui.Utils;
+
+/**
+ * Enum representing the possible Y axes on a spectrum plot.
+ */
+public enum SpectrumYAxisTypes {
+	/* Enum constant indicating the Y axis is a count rate, in counts per microsecond. */
+	COUNT_RATE("Counts (/" + Utils.MU + "s)"),
+	/* Enum constant indicating the Y axis is showing cumulative counts. */
+	ABSOLUTE_COUNTS("Counts");
```

Comment:
Please could the count rate label be changed from "Counts (/mu s)"? The bracket notation indicates the units of the quantity, so we currently have two quantity options `Counts` and `Counts`. The only difference is one has units. I know the count rate option is how it is currently but the new option leads to ambiguity. In the VI they are represented as `Counts/t` and `Counts`, notably no brackets on the former.

Please could you use one of the following for count rate, whichever you think looks best:

- `Count rate (/mu s)`
- `Count rate (mu s)^-1`
- `Counts/t (/mu s)`
- `Counts/mu s`

Personally I like the last one. It scans well, it's VI-like which always make scientists happy, and it's more precise than the existing VI label.